examples/helloworld/swiftExample/swiftExample/AppDelegate.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  AppDelegate.swift
+//  swiftExample
+//
+//  Created by june on 26/01/2019.
+//  Copyright © 2019 june. All rights reserved.
```

Comment:
Please remove this copyright statement that was inserted by Xcode. You can replace it with the Apache v2.0 license that we use for the rest of tink:

```suggestion
/**
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *     http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 *
 **************************************************************************
 */
```

Reply:
Also, please replace it in the other source code locations as well.